rllib/examples/env/rock_paper_scissors.py

```diff
@@ -23,6 +23,9 @@ def __init__(self, config):
         self.last_move = None
         self.num_moves = 0
 
+        # For test-case inspections (compare both players' scores).
```

Comment:
This allows us to inspect for actual learning progress of one of the players (winning over the other). Rewards are always 0.0 (zero-sum game).